src/ImageCarouselReact/widget/components/Carousel.tsx

```diff
@@ -5,116 +5,33 @@
  * This code is under the MIT license found here:
  * https://github.com/react-bootstrap/react-bootstrap/blob/master/LICENSE
  */
-import { IBootstrapProps, prefix } from "../utils/bootstrapUtils";
+import { prefix } from "../utils/bootstrapUtils";
 import ValidComponentChildren from "../utils/ValidComponentChildren";
```

Comment:
Can we make this work without this helper component? As we are not making a framework but a single implementation.